### ISSUE TITLE
Decimal validation should go over Globalize 

### ISSUE THREAD

**Issue by alexander-schranz** [massiveart/husky-validation#57]
### Why?

Because the validation is currently not based on the locale.
### Possible Solution

Globalize valids the input and when the input dont match correctly it will return `NaN`.

``` js
return !isNaN(Globalize.parseFloat(val));
```

Not sure if it is a BC Break. The regExp parameter could also be deleted because currently it dont work because `this.options.regExp` is a string when set the param over aura and will error on `this.options.regExp.test` (`this.options.regExp.test is not a function`).

**Comment by alexander-schranz** [comment]
@wachterjohannes @danrot

**Comment by danrot** [comment]
@wachterjohannes can you have a look?

**Comment by wachterjohannes** [comment]
i dont think it would a be a BC break because as @alexander-schranz said the regexp couldn't be overridden by options. i think we should do it!
